addToCart/style.css

```diff
@@ -0,0 +1,110 @@
+*{
+    margin:0;
+    padding:0;
+    user-select: none;
+}
+:root{
+    --light-orange: #f6c52d;
+    --dark-orange : #f5aa15;
+    --succes-green: #7acd1f;
```

Comment:
I like css variables too but for a more complete compatibility it will be better make classes like .light-orange, dark-orange and apply them to the element

always check https://caniuse.com/#feat=css-variables

Reply:
Done